src/containers/Sensors/Sensor.tsx

```diff
@@ -19,7 +19,10 @@ import { LinearProgress } from 'libs/mui';
 import { prepareSensorTaxonomy } from 'presenters/sensor';
 import { FAQ, FeedbackQuestion, System } from 'common/types';
 import NotFound from 'components/NotFound';
-import { trackEvent } from 'libs/ga';
+import { trackFeedback } from 'libs/analytics';
+import firebase from 'firebase/app';
+
+const timestamp = firebase.firestore.Timestamp.now().seconds;
```

Comment:
Should this be calculated right before or during the addFeedback run? This would timestamp when the page was rendered